src/main/java/com/imagination/cbs/controller/BookingController.java

```diff
@@ -56,5 +61,16 @@
 		BookingDto processedBooking = bookingServiceImpl.processBookingDetails(bookingId, booking);
 		return new ResponseEntity<BookingDto>(processedBooking, HttpStatus.OK);
 	}
+	
 
+	@GetMapping("/draft")
+	public Page<BookingDashBoardDto> findDraftBookings(
+			@RequestParam String status,
+			@RequestParam(defaultValue = "0") Integer pageNo,
+            @RequestParam(defaultValue = "5") Integer pageSize)
+	{
+	//	String loggedInUser = CBSUser.getDisplayName();
+		String loogedInUser = "Mitul";
```

Comment:
Please take off this commented line ... is hardcoded string is necessary ? if yes then mark it final and use as constant.